docs/API.md

```diff
@@ -538,6 +538,7 @@ __minio.PutObjectOptions__
 | `opts.PartSize`                | _uint64_               | Specify a custom part size used for uploading the object                                                                                                                           |
 | `opts.ReplicationVersionID`                | _string_               | Specify VersionID of object to replicate.This option is intended for internal use by MinIO server to extend the replication API implementation by AWS. This option should not be set unless the application is aware of intended use.                                                                                              |
```

Comment:
We should validate that on the client that this is `uuid` - since this is MinIO specific extension.

Reply:
added a validation here.